cost-model/src/transaction_cost.rs

```diff
@@ -11,12 +15,12 @@ use {crate::block_cost_limits, solana_sdk::pubkey::Pubkey};
 const SIMPLE_VOTE_USAGE_COST: u64 = 3428;
 
 #[derive(Debug)]
-pub enum TransactionCost {
-    SimpleVote { writable_accounts: Vec<Pubkey> },
-    Transaction(UsageCostDetails),
+pub enum TransactionCost<'a, Tx: SVMMessage> {
```

Comment:
Was simpler for tests to just make this generic now as well.
See `WritableKeysTransaction` below, which is used in several tests and benches.